docs/mivot/index.rst

```diff
@@ -217,12 +217,12 @@ identifiers, which have the following structure: ``model:a.b``.
     mivot_instance = mivot_viewer.dm_instance
 
     print(mivot_instance.__dict__.keys())
```

Comment:
Unrelated to this PR, but maybe worth considering it for a later docs cleanup:

This is not typical, if these keys are expected to be used by the user, better to exposed them in the API, or reach it through the `dict` property?

However, maybe a rename of that property would be something we should do to avoid any confusion with `dict()`. At least I wondered what it is and why is it there.

(however, I would ping e.g. @andamian if he agrees or not)

Reply:
This snippet was here to show how to use Python object built-in functionalities. 
The method `dict()` actually exist and it is illustrated above in the doc.
I removed the `__dict__` example which has no place in a readme indeed.

Reply:
@lmichel:  can the structure be also inferred by calling `repr()` which seems to return the json representation? Isn't that worth mentioning?

Reply:
> The method dict() actually exist and it is illustrated above in the doc.

Yes, my point was to maybe it better to call it something else? `to_dict()`, or `dictionary()`, etc, just to avoid any confusion with the builtin `dict()` function (I mean, it confused me for a bit)

Reply:
Fixed in the last commit:
- property `dict` replaced with method `to_dict()` 
- property `hk_dict` replaced with method `to_hk_dict()`

Reply:
`repr()` method explicitly mentioned in the doc